statsd/__init__.py

```diff
@@ -1,9 +1,10 @@
 from __future__ import absolute_import
 
-from .client import StatsClient
+from .client import UDPStatsClient
 from .client import TCPStatsClient
-
+from .client import UnixSocketStatsClient
+from .client import StatsClient
 
 VERSION = (3, 2, 1)
```

Comment:
I wonder if this is an upstream bug, not updating the version. hopefully shouldn't bite us

Reply:
when I'm going to merge these in and make the app PR to point to these guys, I'll checkout on an ondemand, buildout and make sure its pulling the right stuff